index.html

```diff
@@ -5,12 +5,41 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Hello microverse</title>
-    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" media="screen and (min-width: 600px)" href="assets/css/smallscreen.css">
+    <link rel="stylesheet" media="screen and (max-width: 600px)" href="assets/css/widecreen.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
 
 </head>
 
 <body>
-    <h1 class='hello-microvers-title'>Hello microverse</h1>
+    <!-- Header -->
+    <div class="header">
+        <nav>
+            <div class="logo"><h1>My logo</h1></div>
+            <div class="menu-icon"><i class="fa-solid fa-bars"></i></div>
+        </nav>
+    </div>
+    <!-- end of header -->
+    <!-- landing page -->
+    <section class="hero">
+        <h1> Hey There.<br>I'm Jenadius, </h1>
+        <h5 >I'm a Software Developer.</h5>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
+        don't hestiate to contact me.</p>
+        <div class="link-container">
+            <a href="#" class="links"><i class="fa-brands fa-github"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-facebook-f"></i></a>
+            <a href="#" class="links"><i class="fa-brands fa-reddit-alien"></i></a>
```

Comment:
- [ ] Please wrap your social media links in a list, You can use `<ul>` list. When we have similar elements in HTML it's a good practice to wrap them in a list and it also makes sure our code is semantic.